### DISCUSSION TITLE
This looks amazing: can my organisation be a test case?

### DISCUSSION THREAD

**Discussion by aeturrell** [microsoft/AzureTRE#3696] [General]
Hi there,

What you're doing here—lowering the barriers to creating a TRE—is really amazing. I work in a large public sector organisation with lots of data (some of which is sensitive), many technically able staff, but (as yet) no cloud solution and we would potentially be interested in being a test case for this TRE solution. Is that something you would be interested in?

**Comment by martysharkey** [comment]
This is an amazing solution for sure, and does lower the barrier to entry significantly. We'd be very happy to discuss how you can get access, as your organisation may have already opted to deploy it. 

Please reach out to Chris Dolan (chris.dolan@cloudkubed.com) or you can contact me directly (martin.sharkey@cloudkubed.com). 

Best regards
Martin

**Reply by marrobi** [reply]
@aeturrell @martysharkey let me know how you get on, as if is already a deployment "where you work", it makes sense to join those dots, if not and need further assistance then happy to try facilitate a wider conversation.

**Comment by CD0lan** [comment]
@aeturrell - I just wanted to follow-up your previous message regarding Azure TRE. Happy to discuss the work we are doing alongside Microsoft in this area. If you can drop me an email with your contact details to chris.dolan@cloudkubed.com I'd be happy to arrange a meeting with you to discuss further. 

Kind regards,

Chris

**Reply by aeturrell** [reply]
Hi Chris,

Thanks so much for getting back to me. I've discussed this with the rest of my org and, for now, we've had to prioritise other new cloud infrastructure. Short story is, would love to explore this but there are just higher priority bits of work to do. But I will most surely get in touch if/when that changes. Thank you for being so responsive and for reaching out, it's massively appreciated.

**Reply by CD0lan** [reply]
Hi Arthur,

Thanks for coming back to me and understand your position.

As and when this changes please don't hesitate to get back in touch. You have my contact details now and I've also sent you a connection request on LinkedIn. We have recently also created an Azure TRE LinkedIn group and I'd like to invite you to that so you will see content that we will be sharing as we progress this amazing offering alongside Microsoft. 

Kind regards,

Chris
